packages/template-react/package.json

```diff
@@ -25,7 +25,9 @@
     "react-router-dom": "^5.0.0"
   },
   "devDependencies": {
-    "jest": "^23.4.2",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^24.8.0",
```

Comment:
I would prefer to remove `babel-jest` here, because we have it already as a dependency in the `jest-preset-hops`:
https://github.com/xing/hops/blob/master/packages/jest-preset/package.json#L31

Also, I would like to keep jest at version 23 for now, because unfortunately we still have some discussion points open, for how we could upgrade to jest 24 without being breaking for our users (see https://github.com/xing/hops/issues/815 and https://github.com/xing/hops/pulls?q=is%3Apr+is%3Aopen+label%3Ablocked)

I will try to raise this topic to our attention and investigate if we can find a solution where we can safely upgrade to jest 24, but until then we should stay at 23 to be on the safe side.

So in a nutshell: It will be enough to just add `babel-core@7.0.0-bridge.0` to the dev dependencies ;)